zip-regular-periodic-turnstiles.rst

```diff
@@ -0,0 +1,141 @@
+::
+
+  ZIP: ???
+  Title: Regular Periodic Turnstiles
+  Owner: Nathan Wilcox <nathan@electriccoin.co>
+  Original-Authors: Nathan Wilcox
+  Status: Draft
+  Category: Protocol Design Policy
+  Created: 2020-01-20
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are to
+be interpreted as described in RFC 2119. [#RFC2119]_
+
+The terms below are to be interpreted as follows:
+
+Sprout protocol
+  Code-name for the Zcash shielded protocol at launch.
+Sapling protocol
+  Code-name for the Zcash shielded protocol added by the second Zcash network
+  upgrade, also known as Network Upgrade 1.
+Blossom protocol
+  Code-name for the Zcash shielded protocol added by the third Zcash network
+  upgrade, also known as Network Upgrade 2.
+
+
+Abstract
+========
+
+This ZIP establishes a protocol design policy to make Zcash `shielded pool turnstiles` a regular process on a predictable schedule. The schedule introduces a new shielded pool once per year, as well as an established `retirement process` for older pools.
+
+Motivation
+==========
+
+…
```

Comment:
@nathan-at-least we cannot merge the ZIP without at least a motivation section filled out. 🙏

Reply:
I have no problem with to creating a new pool when it's technically justified by a change to the shielded protocol. However, I'm highly skeptical of doing so for other upgrades. At the very least, the motivation section should address potential downsides:

* it isn't clear that sufficient privacy can be obtained from the ZIP 308 migration mechanism (which has had little attention to breaking it from the academic community);
* in particular, there is a *major* privacy impact for the first users of a new pool;
* the inconvenience of turnstiles may discourage adoption;
* the protocol becomes significantly more complex, since the same addresses can be used for multiple pools;
* it's not clear that there is any significant security benefit: absent any change to the shielded protocol, any balance violation weakness would simultaneously affect all shielded pools using that version of the protocol.